### PR TITLE
Added Test, POM Page and Folder for Company, changed element in NavBa…

### PR DESCRIPTION
**Pull request by JuliaEde**
…r.js and added el to UsersList.page.js

### CODE REVIEW

**Review by ponomid — APPROVED** [review]
LGTM, good job!

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    before('Create New User', async () => {
+        let email = `${Date.now()}@gmail.com`;
+        let password = 'superUser123!';
+        await createAndLoginAPI(email, password);
+        await LoginPage.login(email, password);
+    })
+
+    it('Should Navigate to Users List', async () => {
+        await PublicationsPage.navBar.btnHamburgerMenu.click();
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:17]: Please use method `await PublicationsPage.navBar.openPeople()`

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        await PublicationsPage.navBar.btnPeople.click();
+        await expect(UsersList.usersPageTitle).toHaveText('users');
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:19]: Please use our NavBar component which has title
`await expect(UsersList.navBar.pageTitle).toHaveText('users')`

File: test/ui/pageobjects/user/UsersList.page.js [hunk 3]
```diff
@@ -11,6 +11,9 @@ class UsersListPage extends Page {
     get userFullNameText() {
         return $(`(//div[@class="text-truncate"])[1]/b`);
     }
+    get usersPageTitle() {
```
Comment (yuliahar) [inline test/ui/pageobjects/user/UsersList.page.js:14]: Please delete it from here. We need to use our component NavBar which has pageTitle element.

File: test/ui/pageobjects/company/Companies.page.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const Page = require('../Page');
+const NavBar = require("../components/NavBar");
+
+class CompaniesPage extends Page {
+
+    navBar = new NavBar();
+
+    // Title
+    get companiesPageTitle() {
```
Comment (yuliahar) [inline test/ui/pageobjects/company/Companies.page.js:9]: Please delete it from here. We need to use our component NavBar which has a pageTitle element.

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    });
+
+    it('Should Navigate to Companies List', async () => {
+        await UsersList.navBar.btnHamburgerMenu.click();
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:23]: Please create a method openCompanies() in the Navbar component which clicks on hamburgerMenu button and then on companies option.

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        await UsersList.navBar.btnCompanies.click();
+        await expect(Companies.companiesPageTitle).toHaveText('companies');
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:25]: Please use our component NavBar:
`await expect(Companies.navBar.pageTitle).toHaveText('companies')`

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 2 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js:19]: Sure, I forgot about it, I thought we only had it for Publications, didn't realize it was for all pages, same element, thank you!

File: test/ui/pageobjects/user/UsersList.page.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/pageobjects/user/UsersList.page.js]: Done

File: test/ui/pageobjects/company/Companies.page.js (hunk 4 above)
Comment (JuliaEde) [reply test/ui/pageobjects/company/Companies.page.js]: Done, deleted

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 1 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js:17]: Done, changed

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 5 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js]: Done, I created openCompanies method in NavBar yesterday, I think, I used it

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 6 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js]: Done, changed

**Review by yuliahar — APPROVED** [review]

**Pull request merged**